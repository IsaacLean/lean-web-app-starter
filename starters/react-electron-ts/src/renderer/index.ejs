<!doctype html>
<html>
    <head>
        <meta charset="utf-8" />
        <title><%= htmlWebpackPlugin.options.title %></title>
        <meta name="viewport" content="width=device-width,initial-scale=1" />
    </head>
    <body>
        <div id="root"></div>
        We are using Node.js <span id="node-version"></span>, Chromium
        <span id="chrome-version"></span>, and Electron
        <span id="electron-version"></span>.
    </body>
</html>
