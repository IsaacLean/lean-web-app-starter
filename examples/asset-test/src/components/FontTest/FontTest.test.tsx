import renderer from "react-test-renderer";

import FontTest from ".";

test("FontTest component matches snapshot", () => {
  const tree = renderer.create(<FontTest />).toJSON();
  expect(tree).toMatchSnapshot();
});
