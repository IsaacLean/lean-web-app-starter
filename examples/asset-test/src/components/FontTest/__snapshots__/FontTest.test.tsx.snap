// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FontTest component matches snapshot 1`] = `
<div
  className="font-test"
>
  <h2>
    Font Test
  </h2>
  <p>
    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed ac justo id dolor molestie pellentesque. Morbi odio nisi, cursus id turpis in, sollicitudin varius urna. Mauris at arcu sit amet quam dignissim tempor non vel lectus. Curabitur convallis orci diam, eu sagittis elit suscipit ut. Phasellus sit amet elit non risus viverra luctus sit amet sed tortor. Maecenas id ultricies justo, eget convallis ex. Pellentesque dapibus dui id facilisis semper.
  </p>
  <p>
    Quisque purus purus, ultrices vel elit nec, pharetra feugiat tortor. Fusce molestie mollis mauris, vitae feugiat sapien suscipit non. Donec pharetra, risus in porttitor consectetur, sapien lectus ultrices massa, quis iaculis tellus justo sit amet est. Morbi volutpat enim et nulla maximus posuere. Phasellus nibh nisi, volutpat sit amet arcu quis, dignissim ultricies leo. Phasellus egestas auctor ante, sed venenatis ligula varius ac. Mauris a finibus sapien, vitae imperdiet nunc. Vestibulum ut enim lacus. Aliquam erat volutpat. Nam purus nisi, sagittis a metus id, pellentesque finibus velit. Etiam efficitur, nunc a lobortis ultrices, turpis enim rhoncus nulla, id rutrum lorem neque non mauris. Sed ut lacinia dui.
  </p>
  <p>
    Sed sagittis odio libero, a rutrum ex tristique eu. Vestibulum eget felis non nulla fermentum pulvinar. Sed ac auctor massa. Integer finibus, nibh eu fringilla vehicula, nisl erat interdum est, in consectetur eros arcu vel metus. Phasellus a velit eget mi suscipit luctus. Morbi venenatis eleifend egestas. Morbi in placerat est, et vulputate ligula. Nulla ullamcorper tincidunt purus non suscipit.
  </p>
</div>
`;
